model/task_lifecycle.go

```diff
@@ -771,6 +771,8 @@ func doBisectStepbackForGeneratedTask(ctx context.Context, generator *task.Task,
 // MarkEnd updates the task as being finished, performs a stepback if necessary, and updates the build status
 func MarkEnd(ctx context.Context, settings *evergreen.Settings, t *task.Task, caller string, finishTime time.Time, detail *apimodels.TaskEndDetail,
 	deactivatePrevious bool) error {
+	ctx, span := tracer.Start(ctx, "mark-end")
```

Comment:
Can we add `TaskExecutionOtelAttribute` to the span here?

Reply:
done